src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -85,19 +85,19 @@ public class Swerve extends SubsystemBase implements Tuneable {
             BACK_LEFT_LOCATION,
             BACK_RIGHT_LOCATION);
 
-    private RotationalSensorHelper gyroYawHelper;
+    private RotationalSensorHelper gyroYawHelperCCW;
 
     public Swerve() {
         fieldsTable.update();
 
-        gyroYawHelper = new RotationalSensorHelper(
-                gyroIO.isConnected.getAsBoolean() ? gyroIO.yawDegreesCW.getAsDouble() : 0);
+        gyroYawHelperCCW = new RotationalSensorHelper(
+                Rotation2d.fromDegrees(gyroIO.isConnected.getAsBoolean() ? -gyroIO.yawDegreesCW.getAsDouble() : 0));
 
         poseEstimator = new SwerveDrivePoseEstimator(
                 swerveKinematics,
-                gyroYawHelper.getRawMeasuredAngleCCW(),
+                gyroYawHelperCCW.getMeasuredAngle(),
```

Comment:
At the end you decided that RAW isn't necessary?

Reply:
yes